### PR TITLE
Created templates for model class. Created new maven module for models and dto.

### PR DESCRIPTION
**Pull request by Mellifaro**

### CODE REVIEW

File: basis/common-model/pom.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>basis</artifactId>
+        <groupId>com.softgroup</groupId>
+        <version>0.0.1</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>common-model</artifactId>
```
Comment (Dominys) [inline basis/common-model/pom.xml:12]: @Mellifaro Why you need this separate module ?

Comment (Mellifaro) [reply basis/common-model/pom.xml:12]: I thought, that all models had to be in a separate module. If it is better to move models to common-router or another module, I will move them. And I have one more question. Should we place all our dao classes for working with database into one separate module?

Comment (Dominys) [reply basis/common-model/pom.xml:12]: @Mellifaro Each module must have it's own DTO.

Comment (Mellifaro) [reply basis/common-model/pom.xml:12]: ok. I am going to create module "common-dao". In this module I want to create package "model" and put all models in it(Contact, Conversation, ConvDetails, ConvSettings, Cursor, Message, Profile, ProfSettings, ProfStatus). All dto (MessageRequestDto and so on) i will put in "auth-api", "profile-api", "messanger-api". Will it be right?

Comment (Mellifaro) [inline basis/common-model/pom.xml]: All dto*

Comment (Dominys) [reply basis/common-model/pom.xml:12]: @Mellifaro Common-dao is module for assessing database. There will be no models and dto in dao module.

**Pull request closed without merging**